x/ecocredit/basket/msg_take_test.go

```diff
@@ -26,7 +26,7 @@ func TestMsgTakeValidateBasic(t *testing.T) {
 			fields: fields{
 				Owner:              addr.String(),
 				BasketDenom:        "BCT",
-				Amount:             "12.34",
+				Amount:             "1234",
```

Comment:
Following the protobuf comments, it seems Amount must be an integer. double-checking here: it that correct?

(if yes, follow-up: should be change the proto field to uint64? or keep string just in case for big ints?)

Reply:
Int but string is the convention with coins

Reply:
OK, so the current PR version does exactly this. resolving